<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestaurantInfoForm</class>
 <widget class="QWidget" name="RestaurantInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>110</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Restaurant Informations</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_RestaurantName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">Restaurant name</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="_label_price">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>44</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spin">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShow">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>75</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>75</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listComment">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCommit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtComment">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>310</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRouting">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>75</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Navigation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Subscribe</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
